humility-doppel/src/lib.rs

```diff
@@ -318,10 +318,29 @@ impl humility::reflect::Load for TaskId {
 pub struct RingbufEntry {
     pub line: u16,
     pub generation: u16,
-    pub count: u32,
+    pub count: RingbufCount,
     pub payload: Value,
 }
 
+#[derive(Clone, Debug)]
+pub struct RingbufCount(pub u32);
```

Comment:
Note that as of https://github.com/oxidecomputer/hubris/pull/1733, ringbuf counts can also be `()` if de-duplication is disabled. This shouldn't currently occur in the wild *yet*, as we aren't actually disabling deduplication anywhere, but we ought to start handling the case where there are no counts, too. I'm fine with doing that separately if you prefer not to do it in this PR.

Reply:
Switched to an `Option<u64>` (also in bdd39928cc7bae70e192085f844f1b7d1b443ab7).  We should get a unit test for this at some point, but this is fine for now IMHO